internal/modules/leverage/test.py

```diff
@@ -0,0 +1,42 @@
+import time
+import logging
+import unittest
+import inspect
+import pathlib
+from internal.modules.gov.tx import submit_and_pass_proposal
+
+from modules.leverage.query import (
+    query_registered_tokens,
+)
+
+logging.basicConfig(format="%(message)s", level=logging.DEBUG)
+
+class TestLeverageModuleTxsQueries(unittest.TestCase):
+    def update_leverage_registry(self):
+        update_registry_path = pathlib.Path().resolve().joinpath("./internal/modules/leverage/update-registry-only-umee.json")
+        submit_and_pass_proposal(
+            proposal_file_or_name=update_registry_path,
+            proposal_type='update-registry',
+            extra_args='200uumee'
+        )
+        time.sleep(10)
+
+    def test_query_total_supply(self):
+        status, res = query_registered_tokens()
+        self.assertTrue(status)
+
+        if len(res['registry']) == 0:
+            self.update_leverage_registry()
+
+        status, res = query_registered_tokens()
+        self.assertTrue(status)
```

Comment:
this is already checked in line 26

Reply:
It was called query_registered_tokens twice